templates/.config/ALZ-Powershell.config.json

```diff
@@ -1,5 +1,10 @@
 {
     "starter_modules": {
+        "platform_landing_zone": {
+            "location": "platform_landing_zone",
+            "short_name": "Preview (Do Not Use for Production) - Azure Verified Modules for ALZ Platform Landing Zone",
```

Comment:
Suggest removing the parenthesis - preview terms are public and can be referenced if necessary